test/functional/fixtures/regression/gh-8054/pages/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>gh-8054</title>
+</head>
+<body>
+<h1>Service worker</h1>
+<button>click me</button>
+<script>
+    const registerServiceWorker = async () => {
+        if ("serviceWorker" in navigator) {
+            try {
+                const serviceWorkerUrl = '/fixtures/regression/gh-8054/pages/sw.js';
+
+                const registration = await navigator.serviceWorker.register(serviceWorkerUrl, {
+                    scope: "/fixtures/regression/gh-8054/pages/",
+                });
+                if (registration.installing) {
```

Comment:
Are you sure that we need these console logs?

Reply:
These logs does not affect the test execution. I can remove it.